### PR TITLE
making a new file

### PR DESCRIPTION
**Pull request by JackFang1998**

### CODE REVIEW

File: JackFang.html [hunk 1]
```diff
@@ -0,0 +1 @@
+hi, This is Jack. Dont cha know!! you betcha!!
```
Comment (stevekindt) [inline JackFang.html:1]: This is a comment

Comment (boychapz) [inline JackFang.html:1]: is this right?

Comment (kelly-bazella) [inline JackFang.html:1]: is this right?

Comment (tommyboone) [reply JackFang.html:1]: Like this?

Comment (JackFang1998) [reply JackFang.html:1]: YES IT IS:)

**Review by stevekindt — APPROVED** [review]

**Review by boychapz — APPROVED** [review]

**Review by kelly-bazella — APPROVED** [review]

**Review by laurenhallau — APPROVED** [review]

**Pull request merged**